infrastructure/template.yaml

```diff
@@ -65,7 +68,50 @@ Globals:
 
 Resources:
   ######################################
-  # Dynamo DB - Output target for system
+  # Raw events store
+  ######################################
+  RawEventsStore:
+    Type: AWS::DynamoDB::Table
+    DeletionPolicy: Delete
+    UpdateReplacePolicy: Delete
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: id
+          AttributeType: S
+        - AttributeName: timestamp
+          AttributeType: N
+      BillingMode: PAY_PER_REQUEST
+      TableName: !Ref RawEventsStoreTableName
+      KeySchema:
+        - AttributeName: id
+          KeyType: HASH
+        - AttributeName: timestamp
+          KeyType: RANGE
+      PointInTimeRecoverySpecification:
+        PointInTimeRecoveryEnabled: true
+      SSESpecification:
+        SSEEnabled: true
+        SSEType: KMS
+        KMSMasterKeyId: !GetAtt DatabaseKmsKey.Arn
+      StreamSpecification:
+        StreamViewType: NEW_IMAGE
+      TimeToLiveSpecification:
+        AttributeName: remove_at
+        Enabled: true
+      Tags:
+        - Key: Product
+          Value: GOV.UK Sign In
```

Comment:
ooh well remembered on the tags, not sure I've been doing that consistently.

Let's work out what we want to tag, could be every resource right?
Or just sites of persistence?

Reply:
We should steal what the Bravo team have done [here](https://github.com/alphagov/di-accounts-infra/blob/main/platform-common/template.yaml#L19-L34) and set the values as params. That'll remove a bit of repetition at least. 

I wonder if you can use the `Globals` thing as well to tag everything?

Reply:
Ah no rubbish.

>The Globals section is supported by the AWS::Serverless::Function, AWS::Serverless::Api, AWS::Serverless::HttpApi, and AWS::Serverless::SimpleTable resources. [[1]]

[1]: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-template-anatomy-globals.html